### PR TITLE
feat!: support type args

### PR DESCRIPTION
**Pull request by renancaraujo**
## Status

READY

## Breaking Changes

Possible tests that were positive before can be flagged as negative now. 

## Description

Add support for type args. More on this: #66

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [x] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Comment by felangel** [comment]
Thanks so much for taking the time to open a PR! I'll review as soon as I can. 🙏

File: packages/mocktail/lib/src/_is_invocation.dart [hunk 1]
```diff
@@ -17,11 +17,14 @@ class _InvocationMatcher implements Matcher {
       }
       return d;
     }
-    // For a method, return <member>(<args>).
-    d = d
-        .add(_symbolToString(invocation.memberName))
-        .add('(')
-        .addAll('', ', ', '', invocation.positionalArguments);
+    // For a method, return <member><<typeergs>>(<args>).
```
Comment (felangel) [inline packages/mocktail/lib/src/_is_invocation.dart:20]:
```suggestion
    // For a method, return <member><<typeargs>>(<args>).
```

**Comment by felangel** [comment]
Updated formatting but it looks like tests are failing now. I'll try to take a look tomorrow.

**Comment by renancaraujo** [comment]
> Updated formatting but it looks like tests are failing now. I'll try to take a look tomorrow.

That was a weird break, something related to the local package dir. Maybe a re-run?

**Comment by felangel** [comment]
@renancaraujo tests are failing with the current implementation. You can run them locally for more information. I'll try to take a closer look soon but haven't had time recently.

**Pull request merged**